watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/amo_1690/amo_1690_case_study.yaml

```
@@ -0,0 +1,52 @@
+base_currency: USD_2020
+base_period: year
+defined_flows:
+  electricity:
+    value: 0.065
+    units: USD_2020/kWh
+  biogas:
+    value: 0.4
+    units: USD_2020/m^3
+  fertilizer:
+    value: 0.5
+    units: USD_2020/kg
+global_parameters:
+  plant_lifetime:
+    value: 20
+    units: year
+  utilization_factor:
+    value: 0.9
```

```suggestion
    value: 0.85
```
Same comment as above

---

0.85 is usually our default, but this team gave us some TEA/LCA data that lists 0.9. So I used that instead.